tee-worker/ts-tests/integration-tests/common/utils/assertion.ts

```diff
@@ -44,38 +45,47 @@ export async function assertFailedEvent(
         );
     }
 }
-
-export async function assertInitialIdGraphCreated(
-    context: IntegrationTestContext,
-    signer: KeyringPair[],
-    events: any[]
-) {
+export async function assertInitialIdGraphCreated(context: IntegrationTestContext, signer: KeyringPair, events: any[]) {
     assert.isAtLeast(events.length, 1, 'Check InitialIDGraph error: events length should be greater than 1');
+
     for (let index = 0; index < events.length; index++) {
         const eventData = events[index].data;
+        const keyringType = signer.type;
 
-        const who = eventData.account.toHex();
+        // evm address should convert to substrate address before compare
+        const who = keyringType === 'ethereum' ? eventData.account.toHuman() : eventData.account.toHex();
```

Comment:
There is function for that `getAddressInSubstrateFormat`, don't need to make it ss58

Reply:
thanks.fixed.